src/app/features/event/event-form/hashtags/hashtags.service.ts

```diff
@@ -0,0 +1,119 @@
+import { ElementRef, inject, Injectable } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import {
+  MatAutocompleteSelectedEvent,
+  MatAutocompleteTrigger,
+} from '@angular/material/autocomplete';
+import { MatChipInputEvent, MatChipEditedEvent } from '@angular/material/chips';
+import { SnackBarService } from '@shared/services/snack-bar.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HashtagsService {
+  private snackBarService = inject(SnackBarService);
+  maxChars = 50;
+  maxHashtagsCount = 100;
```

Comment:
Jako, że to są stałe używane jedynie w serwisie to powinny być prywatne oraz readonly. Dodatkowo można zmienić nazwę na MAX_CHARS oraz MAX_HASHTAGS_COUNT

Reply:
poprawione